sinatra-contrib/lib/sinatra/contrib/version.rb

```diff
@@ -1,6 +1,6 @@
 module Sinatra
   module Contrib
-    VERSION = '2.1.0'
+    VERSION = '2.1.1'
```

Comment:
Let's leave version changes up to the releaser, can you remove this? :bow:

Reply:
Done in b418b2a02205fd84aaeb0901f8f1219c5586c2e6.

I realized later this was not useful but forgot to go back and undo it.